regenerate.cmd

```diff
@@ -0,0 +1,66 @@
+@echo off
```

Comment:
Is `gulp regenerate` still having problems?

Reply:
It is on my desktop. I'm going to install the latest version of node.js and see if that helps.

Reply:
Even with the latest node.js I'm still having issues. I'll merge this pull request and then start working on figuring out what is going on with my machine.